platform/main.roc

```diff
@@ -22,12 +18,10 @@ platform "webserver"
     ]
     packages {}
     imports [
-        Task.{ Task }, # COMMENT FOR GLUE GEN
+        Task.{ Task },
```

Comment:
No there is also a different type for `mainForHost : Request -> ({} -> Response)` instead of the correct `mainForHost : Request -> Task Response []` this is a workaround @bhansconnect provided.